MIDAS Journals.js

```diff
@@ -0,0 +1,122 @@
+{
+	"translatorID": "86c35e86-3f97-4e80-9356-8209c97737c2",
+	"label": "MIDAS Journals",
+	"creator": "Rupert Brooks",
+	"target": "(insight-journal|midasjournal|vtkjournal).org/browse/publication",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsib",
+	"lastUpdated": "2015-11-10 14:10:06"
+}
+
+/*
+   Midas Journal Translator
+   (Includes ITKJournal,InsightJournal,VTKJournal)
+   Copyright (C) 2016 Rupert Brooks
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Affero General Public License for more details.
+
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+function detectWeb(doc, url) {
+	if (url.match("browse/publication")) return "journalArticle";
```

Comment:
this is already guaranteed by the target regexp above. Since the only required field for translation is `item.title`, you probably want to check here that you can get the title.